internal/engine/wire.go

```diff
@@ -20,7 +21,7 @@ import (
 var DeployerBaseWireSet = wire.NewSet(
 	// dockerImageBuilder ( = ImageBuilder)
 	build.DefaultConsole,
```

Comment:
did you also intend to change DefaultConsole to not use os.Stdout, like you did in the original pr?

Reply:
It looks like Console also _has_ to take an `*os.File` and doesn't accept an `io.Writer` >.>